server/aws-lsp-codewhisperer/src/language-server/telemetryService.ts

```diff
@@ -117,8 +159,36 @@ export class TelemetryService extends CodeWhispererServiceToken {
             suggestionReferenceCount: referenceCount,
             generatedLine: generatedLines,
             numberOfRecommendations: session.suggestions.length,
-            perceivedLatencyMilliseconds: perceivedLatencyMilliseconds,
+            perceivedLatencyMilliseconds: timeSinceLastUserModification,
```

Comment:
Is this up to date with latest definition, or it will be changed?

Reply:
For up-to date, I still need to take a look and update accordingly, if required.